app/controllers/users_controller.rb

```diff
@@ -22,9 +20,57 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @user.update user_params
+      flash[:success] = t "updated"
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def index
+    @pagy, @user = pagy User.all, items: Settings.settings.paging_number
+  end
+
+  def destroy
+    if user&.destroy
```

Comment:
```suggestion
    if user.destroy
```
chỗ này đâu cần toán tử an toàn đâu bé